.github/workflows/core.yml

```diff
@@ -277,10 +277,10 @@ jobs:
           auto-activate-base: false
           use-mamba: true
       - name: run tests for flink before 1.15 (exclusive)
-        if: matrix.flink != '115'
-        run: ./mvnw verify -pl flink/flink-scala-2.11,flink/flink-scala-2.12,flink-cmd,zeppelin-interpreter-integration -am -Pflink-${{ matrix.flink }} -Phadoop2 -Pintegration -DfailIfNoTests=false -Dtest=org.apache.zeppelin.flink.*Test,FlinkIntegrationTest${{ matrix.flink }} ${MAVEN_ARGS}
-      - name: run tests for flink before 1.15 (inclusive)
-        if: matrix.flink == '115'
+        if: matrix.flink < '115'
+        run: ./mvnw verify -pl flink/flink-scala-2.11,flink/flink-scala-2.12,flink-cmd,zeppelin-interpreter-integration -Pflink-${{ matrix.flink }} -Phadoop2 -Pintegration -DfailIfNoTests=false -B -Dtest=org.apache.zeppelin.flink.*Test,FlinkIntegrationTest${{ matrix.flink }} ${MAVEN_ARGS}
```

Comment:
`-B` is part of `${MAVEN_ARGS} it is therefore not necessary at this point.`

Reply:
Thanks @Reamer , I have removed it.